Snabble/UI/Onboarding/Views/OnboardingButtonView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  OnboardingButtonView.swift
+//  Snabble
+//
+//  Created by Uwe Tilemann on 06.08.22.
+//
+//  Copyright © 2022 snabble. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+/// View to render one onboarding item
+struct OnboardingButtonView: View {
+    var item: OnboardingItem
+    var numberOfPages: Int
+    @Environment(\.presentationMode) var presentationMode
+
+    @Binding var currentPage: Int
+
+    @ViewBuilder
+    var leftButton: some View {
+        if let text = item.prevButtonTitle {
+            Button(action: {
+                if currentPage > 0 {
+                    currentPage -= 1
+                }
+                print("right clicked")
+            }) {
+                Text(key: text)
+            }
+            .buttonStyle(.automatic)
+            .buttonStyle(AccentButtonStyle())
+        }
+        EmptyView()
+    }
+
+    @ViewBuilder
+    var rightButton: some View {
+        if let text = item.nextButtonTitle {
+            Button(action: {
+                print("right clicked")
```

Comment:
Willst du `prints` in der Release Build enthalten haben? Ich würde sie entfernen und wenn wir sie wollen, dann mit einem Build Flag arbeiten das sie nur in Debug Build enthalten sind.

Reply:
37cd2231808aa4bd8dce3bff9335072fb3592c51